main.go

```
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/cloudflare/cloudflare-go"
+	csbouncer "github.com/crowdsecurity/go-cs-bouncer"
+)
+
+func deleteExistingCrowdSecIpList(ctx context.Context, api *cloudflare.API) {
+	ipLists, _ := api.ListIPLists(ctx)
+	id, err := getCrowdSecIPListId(ipLists)
+	if err != nil {
+		return
+	}
+
+	resp, err := api.DeleteIPList(ctx, id)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Printf("%+v", resp)
+
+}
+
+func getCrowdSecIPListId(ipLists []cloudflare.IPList) (string, error) {
+	for _, ipList := range ipLists {
+		if ipList.Name == "crowdsec" {
+			return ipList.ID, nil
+		}
+	}
+	return "", errors.New("crowdsec ip list not found")
+}
+
+func main() {
+
+	ctx := context.Background()
+	conf, err := NewConfig("./config.yaml")
+	if err != nil {
+		log.Fatal(err)
+	}
+	cfApi, _ := cloudflare.NewWithAPIToken(conf.CloudflareAPIToken, cloudflare.UsingAccount(conf.CloudflareAccountID))
```

please don't ignore err